pkg/models/devops/devops.go

```diff
@@ -242,7 +242,25 @@ func (d devopsOperator) ListCredentialObj(workspace string, projectName string)
 	if err != nil {
 		return nil, err
 	}
-	return d.k8sclient.CoreV1().Secrets(projectObj.Status.AdminNamespace).List(metav1.ListOptions{})
+	credentialList, err := d.k8sclient.CoreV1().Secrets(projectObj.Status.AdminNamespace).List(metav1.ListOptions{})
```

Comment:
using lister instead will be faster, normally we use client only when update needed

Reply:
I will improve it in next pr，for devops/pipelines/credentials.

Reply:
please fix it now to keep a clean commit log

Reply:
fixed